### PR TITLE
Cef upgrade915

### PR DESCRIPTION
**Pull request by vsadda**

### CODE REVIEW

File: minuet/symphony/Paragon.Extensibility/IApplicationWindow.cs [hunk 1]
```diff
@@ -43,7 +46,7 @@ public interface IApplicationWindow
 
         void Restore();
 
-        void DrawAttention(bool autoclear, int maxFlashes, int timeOut);
```
Comment (lneir) [inline minuet/symphony/Paragon.Extensibility/IApplicationWindow.cs:46]: There are changes in Symphony.sln that need this interface.  We needed this so we could customize flashing icon behavior from pgx.  This feature requested by our PM.  Please take a look.

File: minuet/symphony/Paragon.Plugins.Notifications/Models/NotificationCollection.cs [hunk 2]
```diff
@@ -9,7 +9,7 @@ namespace Paragon.Plugins.Notifications.Models
 {
     public class NotificationCollection
     {
-        private const int MaxNotifications = 11;
+        private const int MaxNotifications = 5;
```
Comment (lneir) [inline minuet/symphony/Paragon.Plugins.Notifications/Models/NotificationCollection.cs:12]: we had changes to bump up to 11.  can we keep this?

File: minuet/symphony/Paragon.Plugins.Notifications/Models/NotificationFactory.cs [hunk 3]
```diff
@@ -50,35 +49,23 @@ private static string GetId(string id)
 
         private static SolidColorBrush ConvertToColor(string text)
         {
-            SolidColorBrush fallback = new SolidColorBrush(Colors.Black);
-
             if (string.IsNullOrEmpty(text))
-                return fallback;
+            {
```
Comment (lneir) [inline minuet/symphony/Paragon.Plugins.Notifications/Models/NotificationFactory.cs:53]: changes at right fix crash issue we found.  
see: https://github.com/SymphonyOSF/SFE-Minuet-DesktopClient/commit/07041d41956d5de191874f21079767e8b276a9da

please keep these changes.

File: minuet/symphony/Paragon.Plugins.Notifications/Models/NotificationCollection.cs [hunk 4]
```diff
@@ -36,7 +36,7 @@ public void Add(Notification notification)
             {
                 var activeNotifications = ActiveNotifications();
 
-                if (activeNotifications.Any())
+                if (!activeNotifications.Any())
```
Comment (lneir) [inline minuet/symphony/Paragon.Plugins.Notifications/Models/NotificationCollection.cs:39]: change above and here from PR: https://github.com/SymphonyOSF/SFE-Minuet-DesktopClient/commit/c4558a75c2ab5b23ea76ba2e62b20635ae9f3aac

can we keep these changes?

File: minuet/symphony/Paragon.Runtime/Kernel/Applications/ApplicationMetadata.cs [hunk 5]
```diff
@@ -38,12 +43,44 @@ public ApplicationMetadata()
...
+            }
+            set
+            {
+                if( value != ApplicationEnvironment.Production )
+                {
+                    try
+                    {
+#if ENFORCE_PACKAGE_SECURITY
+                        // Only if the current user has developer access, we allow non-production evironment to be set
+                        using( RegistryKey key = Registry.CurrentUser.OpenSubKey("Software\\Policies\\Goldman Sachs\\DashNative\\DevelperAccess") )
```
Comment (lneir) [inline minuet/symphony/Paragon.Runtime/Kernel/Applications/ApplicationMetadata.cs:66]: can we avoid GS specific paths?

File: minuet/symphony/Paragon.Runtime/WPF/ParagonWindow.cs [hunk 6]
```diff
@@ -164,9 +171,9 @@ public void StartDragMove()
         }
 
         [SecurityPermission(SecurityAction.LinkDemand, Flags = SecurityPermissionFlag.UnmanagedCode)]
-        public void Flash(bool clear = false, bool autoclear = false, int maxFlashes = 5, int timeOut = 0)
+        public void Flash(bool clear = false, bool autoclear = false)
```
Comment (lneir) [inline minuet/symphony/Paragon.Runtime/WPF/ParagonWindow.cs:174]: this was changed as part of PR: https://github.com/SymphonyOSF/SFE-Minuet-DesktopClient/commit/8c194df02f3640b96bc2d5fed97e1aac12acc310

please keep

**Comment by lneir** [comment]
also, can you provide list of new features?  want to make sure testing hits them.

**Comment by vsadda** [comment]
The most significant new feature is code signing and that has to be enabled through conditional compilation directive.

File: minuet/symphony/Paragon.Runtime/Kernel/Applications/ApplicationMetadata.cs (hunk 5 above)
Comment (vsadda) [inline minuet/symphony/Paragon.Runtime/Kernel/Applications/ApplicationMetadata.cs:66]: This feature is conditionally compiled as you can see to control a GS specific feature (at this point).

Having said that, it probably is a good idea to change the path and remove Goldman Sachs, as this feature is likely be used everywhere in future.

**Pull request merged**